src/clj/rems/application/process_managers.clj

```diff
@@ -0,0 +1,44 @@
+(ns rems.application.process-managers
```

Comment:
some inconsistent namespace referring here:

```clojure
;; alias on line 20
(blacklist/add-user-to-blacklist! ...)

;; full ns on line 34
(rems.api.services.attachment/get-attachments-in-use application)
``` 

would it make sense to always refer by full namespace when using functions outside this namespace? (with allowed exceptions, like `clojure.core`). there's verbosity included but reading and using is simplified quite a bit.